### PR TITLE
splunk-bash

### PR DESCRIPTION
**Pull request by bshuler**
Added splunk-bash as a command to entrypoint.sh

To get a splunk-centric bash prompt:
docker exec -it splunk entrypoint.sh splunk-bash

### CODE REVIEW

File: universalforwarder/entrypoint.sh [hunk 1]
```diff
@@ -121,6 +121,10 @@ EOF
 
   sudo -HEu ${SPLUNK_USER} tail -n 0 -f ${SPLUNK_HOME}/var/log/splunk/splunkd_stderr.log &
   wait
+elif [ "$1" = 'splunk-bash' ]; then
+  source /opt/splunk/bin/setSplunkEnv
+  cd $SPLUNK_HOME
+  su splunk
```
Comment (outcoldman) [inline universalforwarder/entrypoint.sh:127]: `su $SPLUNK_USER`

File: universalforwarder/Dockerfile [hunk 2]
```diff
@@ -37,7 +37,12 @@ RUN apt-get install -y wget sudo \
     && rm -fR ${SPLUNK_HOME}/etc \
     && chown -R ${SPLUNK_USER}:${SPLUNK_GROUP} ${SPLUNK_HOME} \
     && chown -R ${SPLUNK_USER}:${SPLUNK_GROUP} ${SPLUNK_BACKUP_DEFAULT_ETC} \
+<<<<<<< HEAD
+    && rm -rf /var/lib/apt/lists/* \
+    && echo source /opt/splunk/bin/setSplunkEnv >> /etc/bash.bashrc
+=======
     && rm -rf /var/lib/apt/lists/*
+>>>>>>> parent of 177dfeb... Bashify
```
Comment (outcoldman) [inline universalforwarder/Dockerfile:45]: looks like something left after merge

File: universalforwarder/Dockerfile [hunk 3]
```diff
@@ -14,7 +14,7 @@ ENV SPLUNK_BACKUP_DEFAULT_ETC /var/opt/splunk
 
 # add splunk:splunk user
 RUN groupadd -r ${SPLUNK_GROUP} \
-    && useradd -r -m -g ${SPLUNK_GROUP} ${SPLUNK_USER}
+    && useradd -r -m -s /bin/bash -g ${SPLUNK_GROUP} ${SPLUNK_USER}
```
Comment (outcoldman) [inline universalforwarder/Dockerfile:17]: do we still need to set default shell?

**Comment by bshuler** [comment]
Dang it. Better one coming.

**Pull request closed without merging**